beacon-chain/blockchain/execution_engine.go

```diff
@@ -70,7 +70,6 @@ func (s *Service) notifyForkchoiceUpdate(ctx context.Context, arg *notifyForkcho
 
 	nextSlot := s.CurrentSlot() + 1 // Cache payload ID for next slot proposer.
 	hasAttr, attr, proposerId := s.getPayloadAttribute(ctx, arg.headState, nextSlot)
-
```

Comment:
Why remove these lines?

Reply:
will add it back in, was reverting some changes removed extra lines